homeassistant/components/twitch/sensor.py

```diff
@@ -96,10 +97,7 @@ def entity_picture(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        attr = {
-            ATTR_FRIENDLY_NAME: self._channel.display_name,
-        }
-        attr.update(self._statistics)
+        attr = copy.deepcopy(self._statistics)
```

Comment:
You don't need to do a deepcopy, you're only updating the outer level. Shallow copy is enough.
```suggestion
        attr = dict(self._statistics)
```

Reply:
👍